metis-indexing/metis-indexing-client/src/main/resources/indexing.properties.example

```diff
@@ -0,0 +1,12 @@
+mongo.host=host1,...,hostn

```

Comment:
Does this mean that we need an entry in the .gitignore of this project for the file indexing.properties?

Reply:
Correct. Fixed.